modules/SpyUtils.js

```diff
@@ -27,7 +27,11 @@ export const createSpy = (fn, restore = noop) => {
 
   let targetFn, thrownValue, returnValue
 
-  const spy = function spy() {
+  const spy = new Function('spy', `return function(${ // eslint-disable-line no-new-func
+    Array(...Array(fn.length)).map((_, i) => '_'.repeat(i + 1)).join(',')
```

Comment:
this creates an array like `[_, __, ___]` with the same length as the original function